docs/howto/hooks/lua.md

```diff
@@ -373,8 +373,17 @@ Creates a new Delta Lake client used to interact with the lakeFS server.
 ### `formats/delta_client.get_table(repository_id, reference_id, prefix)`
 
 Returns a representation of a Delta Lake table under the given repository, reference, and prefix.
-The format of the response is a table `{number, {string}}` where `number` is a version in the Delta Log, and the mapped 
-`{string}` table (list) contains JSON strings of the different Delta Lake log operations listed in the mapped version entry. 
+The format of the response is two tables: 
+- the first is a table of the format `{number, {string}}` where `number` is a version in the Delta Log, and the mapped `{string}` 
+table (list) contains JSON strings of the different Delta Lake log operations listed in the mapped version entry.
+- the second is a table of the metadata of the table, it consists of the following fields:
```

Comment:
Isn't the metadata versioned as well?
I find it confusing that on the one hand we provide a list of versions for the table and on the other hand we provide only the metadata of the current snapshot. Perhaps a clarification is needed

Reply:
It's the latest version's metadata.
The intention is to make it easier to get the current state of the table to initialize it in other systems faster.
I'll clarify